lib/gltfPipeline.js

```diff
@@ -26,13 +29,24 @@ module.exports = {
     processFileToDisk : processFileToDisk
 };
 
-function processJSON(gltf, options, callback) {
+function writeSources(gltf, outputPath, options, callback) {
+    var embed = options.embed;
+    var embedImage = options.embedImage;
+    var basePath = path.dirname(outputPath);
+    async.each(['buffers', 'images', 'shaders'], function(name) {
+        writeSource(gltf, basePath, name, embed, embedImage, callback);
+    });
+}
+
+function processJSON(gltf, outputPath, options, callback) {
```

Comment:
`outputPath` doesn't really make sense as an argument for `processJSON` since all we want is JSON back. It looks like it was added so that `writeSources` would work, but instead `writeSources` could just always set `embed` to true.